core/dbt/include/README.md

```diff
@@ -1 +1,10 @@
-# Include README
+# Include Module
+
+contains the a few key default `dbt-core` sub directories inlcuding.
+
+- `global_project` which defines the default implementations of jinja2 macros for `dbt-core ` which can be overwritten in each adapter repo to work more in line with those adapter plugins. to view adapter specific jinja2 changes please check the relevant adapter repo.
+
+- `starter_project` which produces the default init project inside the `dbt-core` project after doing the `make dev` command
```

Comment:
The `starter_project` is what we use to generate a project from `dbt init`.  I think it's important to point out that cloud does *not* use this.  They use a tag of https://github.com/dbt-labs/dbt-starter-project.

How is `make dev` using this?